pkg/archive/archive_windows.go

```diff
@@ -3,9 +3,26 @@
 package archive
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar"
 )
 
+// canonicalTarNameForPath returns platform-specific filepath
+// to canonical posix-style path for tar archival. p is relative
+// path.
+func canonicalTarNameForPath(p string) (string, error) {
+	// windows: convert windows style relative path with backslashes
+	// into forward slashes. since windows does not allow '/' or '\'
+	// in file names, it is mostly safe to replace however we must
+	// check just in case
+	if strings.Contains(p, "/") {
```

Comment:
@LK4D4 you can't. http://stackoverflow.com/a/10708449/54929
but as a precautionary measure, I check for "/" and if it's there we just fail rather than blindly replacing it.